Makefile

```diff
@@ -11,7 +11,7 @@ PG_MAJOR?=16
 PG_VERSIONS?=
 
 # Additional PostgreSQL extensions we want to include with specific version/commit tags
-PGAI_VERSION?=v0.3.0
+PGAI_VERSION?=extension-0.4.0
```

Comment:
Why isn't there a v0.4.0 tag? That would remove the need for a lot of this PR, and would keep it more in-line with how things are usually done?

Reply:
the pgai repo is now a mono-repo with both an extension and a python library. the python library is being tagged independently as `pgai-x.x.x` and the extension as `extension-x.x.x`